<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewConnecion</class>
 <widget class="QDialog" name="addNewConnecion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 74, 106);</string>
  </property>
  <widget class="QComboBox" name="connectTypeBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="selectedName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="connectNameBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>增加关系设定类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>增加关系名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddConnection">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="connectionNote">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>具体关系：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
